### ISSUE TITLE
v0.3 ePubs without cover image can not be displayed

### ISSUE THREAD

**Issue by timfreiheit** [futurepress/epub.js#456]
ePubs without an "cover-image" item can not be displayed.
It results in the following error messages:

```
"invalid input", source: http://localhost:8081/epubjsandroid/epubjs_0_3/epubjs/build/epub.js (4444)
"invalid input", source: http://localhost:8081/epubjsandroid/epubjs_0_3/epubjs/build/epub.js (4584)
"invalid input", source: http://localhost:8081/epubjsandroid/epubjs_0_3/epubjs/build/epub.js (4584)
"invalid input", source: http://localhost:8081/epubjsandroid/epubjs_0_3/epubjs/build/epub.js (4584)
"TypeError: invalid input", source: http://localhost:8081/epubjsandroid/epubjs_0_3/epubjs/build/epub.js (4576)
```

the workaround for now is to make sure the ePub contains a dummy cover image item:

``` xml
<item id="cover-image" properties="cover-image" href="DUMMY.jpg" media-type="image/jpeg"/>
```

**Comment by geek1011** [comment]
I do not have this problem.

**Comment by samford** [comment]
I'm experiencing the same issue when using an ePub without a cover image (on an up-to-date v0.3 build).  As above, adding a cover image resolved the issue for the mean time.

**Comment by fchasen** [comment]
I'm fixing the opf parsing now so I should have this fixed in v0.3 asap.

**Comment by fchasen** [comment]
Fixed in v0.3.6 thanks to @gerhardsletten

https://github.com/futurepress/epub.js/pull/519
